django-stubs/contrib/gis/db/models/__init__.pyi

```diff
@@ -9,3 +9,121 @@ from django.contrib.gis.db.models.fields import PointField as PointField
 from django.contrib.gis.db.models.fields import PolygonField as PolygonField
 from django.contrib.gis.db.models.fields import RasterField as RasterField
 from django.db.models import *
+
+__all__ = [
+    "Aggregate",
+    "Avg",
+    "Count",
+    "Max",
+    "Min",
+    "StdDev",
+    "Sum",
+    "Variance",
+    "BaseConstraint",
+    "CheckConstraint",
+    "Deferrable",
+    "UniqueConstraint",
+    "Choices",
+    "IntegerChoices",
+    "TextChoices",
+    "AutoField",
+    "BLANK_CHOICE_DASH",
+    "BigAutoField",
+    "BigIntegerField",
+    "BinaryField",
+    "BooleanField",
+    "CharField",
+    "CommaSeparatedIntegerField",
+    "DateField",
+    "DateTimeField",
+    "DecimalField",
+    "DurationField",
+    "EmailField",
+    "Empty",
+    "Field",
+    "FilePathField",
+    "FloatField",
+    "GenericIPAddressField",
+    "IPAddressField",
+    "IntegerField",
+    "NOT_PROVIDED",
+    "NullBooleanField",
+    "PositiveBigIntegerField",
+    "PositiveIntegerField",
+    "PositiveSmallIntegerField",
+    "SlugField",
+    "SmallAutoField",
+    "SmallIntegerField",
+    "TextField",
+    "TimeField",
+    "URLField",
+    "UUIDField",
+    "Index",
+    "ObjectDoesNotExist",
+    "signals",
+    "CASCADE",
+    "DO_NOTHING",
+    "PROTECT",
+    "RESTRICT",
+    "SET",
+    "SET_DEFAULT",
+    "SET_NULL",
+    "ProtectedError",
+    "RestrictedError",
+    "Case",
+    "CompositePrimaryKey",
```

Comment:
This is also missing: https://github.com/typeddjango/django-stubs/blob/8fd22efc1c91dc3e747665820f594ccd36bd0423/scripts/stubtest/allowlist_todo_django52.txt#L12